units/src/amount/unsigned.rs

```diff
@@ -25,13 +25,13 @@ use super::{
 /// Warning!
 ///
 /// This type implements several arithmetic operations from [`core::ops`].
-/// To prevent errors due to overflow or underflow when using these operations,
-/// it is advised to instead use the checked arithmetic methods whose names
-/// start with `checked_`. The operations from [`core::ops`] that [`Amount`]
-/// implements will panic when overflow or underflow occurs. Also note that
-/// since the internal representation of amounts is unsigned, subtracting below
-/// zero is considered an underflow and will cause a panic if you're not using
-/// the checked arithmetic methods.
+/// To prevent overflow errors when using these operations, it is advised to
+/// instead use the checked arithmetic methods whose names start with
+/// `checked_`. The operations from [`core::ops`] that [`Amount`] implements
+/// will panic when an overflow occurs. Also note that since the internal
+/// representation of amounts is unsigned, subtracting below zero is
+/// considered an overflow and will cause a panic if you're not using the
+/// checked arithmetic methods.
```

Comment:
Oh, this will either conflict with the monadic handling or wasn't addressed by it and the entire section needs to be rewritten to explain it.

Reply:
Good point.  Can't this entire section starting with `Warning!` be removed now that the operations will error instead of panic with monadic handling?  If it's a lot of rewording might just create an issue on that for now.

Reply:
Yes, it can. But I'd love to have some top-level section somewhere explaining it with examples.

Reply:
> Yes, it can.

Done

> But I'd love to have some top-level section somewhere explaining it with examples.

I see you opened https://github.com/rust-bitcoin/rust-bitcoin/issues/4222 so no need to do that here.